pkg/i18n/english.go

```diff
@@ -1149,6 +1149,12 @@ func addEnglish(i18nObject *i18n.Bundle) error {
 		}, &i18n.Message{
 			ID:    "copyBranchNameToClipboard",
 			Other: "copy branch name to clipboard",
+		}, &i18n.Message{
+			ID:    "copyFileNameToClipboard",
+			Other: "Copy the file name to the clipboard",
```

Comment:
```suggestion
			Other: "copy the file name to the clipboard",
```
descriptions for keybindings are conventionally in lower-case :)